<?xml version="1.0"?>
<launch>

    <!-- tell reflex_interface that we're running on real robot hand -->
    <arg name="use_sim_data_hand" default="false" />
    <param name="use_sim_data_hand" value="$(arg use_sim_data_hand)" />
    <arg name="use_sim_data_obj" default="false" />
    <param name="use_sim_data_obj" value="$(arg use_sim_data_obj)" />

    <!-- Keyboard teleoperation node for fingers and wrist (TODO be careful that you're not sending wrist position to UR5!)-->
    <node pkg="reflex_interface" type="finger_teleop_node" name="finger_teleop_node" output="screen"/>

    <!-- Run reflex interface node -->
    <node pkg="reflex_interface" type="reflex_interface_node" name="reflex_interface_node" output="screen"/>

</launch>
